src/sentry/static/sentry/app/views/issueList/noGroupsHandler/index.tsx

```diff
@@ -85,15 +85,16 @@ class NoGroupsHandler extends React.Component<Props, State> {
   }
 
   renderAwaitingEvents(projects: State['firstEventProjects']) {
-    const {organization} = this.props;
-    const project = projects && projects.length > 0 ? projects[0] : null;
+    const {organization, groupIds} = this.props;
+
+    const project = projects && projects.length > 0 ? projects[0] : undefined;
+    const sampleIssueId = groupIds.length > 0 ? groupIds[0] : undefined;
 
-    const sampleIssueId = this.props.groupIds.length > 0 ? this.props.groupIds[0] : '';
     return (
       <ErrorRobot
         org={organization}
         project={project}
-        sampleIssueId={sampleIssueId}
+        sampleIssueId={sampleIssueId?.toString() ?? ''}
```

Comment:
Had to finess things here a bit. Ensures if the sample ID isn't available we set to `''` to indicate there is no first event.